content/source/docs/enterprise/admin/update-tfe-license.html.md

```diff
@@ -136,23 +136,23 @@ In the Airgap installation, the license update operation may finish without any
 
 ### Unable to sync license: Error: Unsuccessful HTTP response
 
-The error message below may occur due to several possible issues.
+Terraform can produce this error message for several different issues.
 
     ```plaintext
     Unable to sync license: Error: Unsuccessful HTTP response
     ```
 
 *   Network communication to the Replicated servers
 
-    This symptom can happen with the online installation type when the current network infrastructure has changed since the installation of Terraform Enterprise. Terraform Enterprise requires network communication to the Replicated endpoints, as noted in the [network requirements](https://www.terraform.io/docs/enterprise/before-installing/network-requirements.html) guideline. With the license sync operation, the Replicated installer will attempt to contact `api.replicated.com` in order to retrieve the license information. The specific IP addresses of Replicated services for the Terraform Enterprise can be found in [Replicated’s GitHub repository](https://github.com/replicatedhq/ips/blob/master/ip_addresses.json).
+    Terraform Enterprise requires network communication to the Replicated endpoints, as noted in the [network requirements](https://www.terraform.io/docs/enterprise/before-installing/network-requirements.html) guideline. During the license sync operation, the Replicated installer attempts to contact `api.replicated.com` to retrieve the license information. In the online installation type, you might see the above error if the network infrastructure changed after you installed Terraform Enterprise. The specific IP addresses of Replicated services for the Terraform Enterprise can be found in [Replicated’s GitHub repository](https://github.com/replicatedhq/ips/blob/master/ip_addresses.json).
 
-*   Mismatched customer name between the installed license and the new license.
+*   Customer name on new license does not match existing license
 
-    This symptom can happen with the Airgap installation type in some conditions, for example, when a trial license was issued with a customer name and the new paid license was issued on a different customer name. If you are on the Airgap installation and encountered this error, please contact [HashiCorp Support](https://www.hashicorp.com/technical-support-services-and-policies) together with the output from the command `replicatedctl license inspect` from the existing Terraform Enterprise host machine to request for further assistance.
+    This may happen with the Airgap installation type for a variety of reasons. For example, when a trial license was issued with a customer name and the new paid license was issued on a different customer name.  Please contact [HashiCorp Support](https://www.hashicorp.com/technical-support-services-and-policies) for assistance. In your request, include the output from running`replicatedctl license inspect` on the Terraform Enterprise host machine.
```

Comment:
```suggestion
    This may occur with an Airgap installation for a variety of reasons. One example is when a trial license is associated with a different customer name from the subsequent paid license.  Please contact [HashiCorp Support](https://www.hashicorp.com/technical-support-services-and-policies) for assistance. In your request, include the output from running`replicatedctl license inspect` on the Terraform Enterprise host machine.
```